micro-docker/README.md

```diff
@@ -1,4 +1,53 @@
-- cd micro-docker
-- sudo docker build -t rackhd/micro .
-- sudo docker save rackhd/micro | xz -z > discovery.docker.tar.xz
-- copy discovery.docker.tar.xz to on-http/static/http/common
+# Micro Docker
+
+## RancherOS vmlinuz
+
+```
+wget https://github.com/rancher/os/releases/download/v1.0.2/vmlinuz -O vmlinuz-1.0.2-rancher
+copy vmlinuz-1.0.2-rancher to on-http/static/http/common
+```
+
+## RancherOS initrd
+
+```
+wget https://github.com/rancher/os/releases/download/v1.0.2/initrd -O initrd-1.0.2-rancher
+copy initrd-1.0.2-rancher to on-http/static/http/common
+```
+
+## Discovery Image
+
+```
+cd discovery
+sudo docker build -t rackhd/micro .
+sudo docker save rackhd/micro | xz -z > discovery.docker.tar.xz
```

Comment:
If you provide node version as an argument, you'd better to add it here as well.